morecantile/models.py

```diff
@@ -300,6 +300,13 @@ class TileMatrix(BaseModel, extra="forbid"):
         Field(description="Describes the rows that has variable matrix width"),
     ] = None
 
+    renderingPixelSize: Annotated[
+        float,
+        Field(
+            description="The physical size (in meters) of a pixel when rendered on a screen"
+        ),
+    ] = 0.28e-3
+
```

Comment:
`renderingPixelSize` is not part of the OGC specification for the TileMatrix object but we could resolve it by using `private variable like https://github.com/developmentseed/morecantile/blob/main/morecantile/models.py#L384-L387